docs/development.rst

```diff
@@ -388,6 +388,14 @@ recommended process for generating a new release is described here.
 
         python setup.py sdist upload
 
+10. Ensure that a built Docker image is tagged on
+    `Girder's Docker Hub page <https://hub.docker.com/r/girder/girder/tags/>`_.
+    Such a build can be added by following the instructions for
+    `adding and running a new build <https://docs.docker.com/docker-hub/builds/#add-and-run-a-new-build>`_.
```

Comment:
A couple of thoughts on that:

1) This could result in different dependencies based on the users image cache. This could be solved by adding the `--no-cache=true` flag to the build instructions.
2) Users could potentially leak the contents/configs of their (dirty) working tree into the built/distributed image.

Generally, I would prefer CI machines building the images rather than one of us. That being said, I've definitely exceeded my bikeshedding allowance recently... so I'll fold pretty easily.